canonical-web-and-design/juju.is
# Report side navigation

## Done
Added accordions and scroll spy to side navigation

## QA
- Go to https://juju-is-299.demos.haus/cloud-native-kubernetes-usage-report-2021
- Click the headings in the side nav and check that they toggle their menus
- Scroll in the page and check that the section is highlighted in the navigation
- **Note**: The accuracy of this depends on the length of each section and how fast you scroll

## Issue
Fixes #300
